server/model/promo.js

```diff
@@ -0,0 +1,11 @@
+import {Record} from 'immutable';
+
+export const Promo = Record({
+  modifiers: null,
+  url: null,
+  image: null,
+  meta: null,
+  title: null,
+  copy: null,
+  type: null
+});
```

Comment:
I like this idea – would we incur any performance cost from passing in the entire `Article` object when we only need a few pieces of it?

And will we treat e.g. podcasts, videos etc. as types of article?